docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.6"
+services:
+  blueoil:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile
+      # https://docs.nvidia.com/deeplearning/frameworks/user-guide/index.html#run_tf_cifar10
+      shm_size: "1gb"
```

Comment:
In default, docker image name will be `blueoil_blueoil`.
Following current `blueoil.sh`, it is better to specify image name for the user by adding `image: blueoil_${USER}` after `build:` part.
https://docs.docker.com/compose/compose-file/
We can use image cache created by `make build` with same docker image name. 👍

Reply:
Marvelous!! thanks!!

Reply:
@suttang 
Oh, I misunderstood, the image cache didn't work as well 😢
I am not sure why the cache doesn't work, I need more investigation, but I think this cache problem is not critical for this PR. So we can consider after this PR 😄

Reply:
@hadusam Thank you!!
Updated!! Check Onegai-shimasu!!